### PR TITLE
Refactor test for easier reuse.

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst or @toregge PR

### CODE REVIEW

**Review by toregge — APPROVED** [review]

File: searchcore/src/tests/proton/documentdb/documentbucketmover/bucketmover_common.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline searchcore/src/tests/proton/documentdb/documentbucketmover/bucketmover_common.cpp:1]: Please update copyright.

File: searchcore/src/tests/proton/documentdb/documentbucketmover/bucketmover_common.h [hunk 2]
```diff
@@ -0,0 +1,131 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline searchcore/src/tests/proton/documentdb/documentbucketmover/bucketmover_common.h:1]: Please update copyright.

File: searchcore/src/tests/proton/documentdb/documentbucketmover/documentmover_test.cpp [hunk 3]
```diff
@@ -0,0 +1,140 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline searchcore/src/tests/proton/documentdb/documentbucketmover/documentmover_test.cpp:1]: Please update copyright.

File: searchcore/src/tests/proton/documentdb/documentbucketmover/scaniterator_test.cpp [hunk 4]
```diff
@@ -0,0 +1,196 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline searchcore/src/tests/proton/documentdb/documentbucketmover/scaniterator_test.cpp:1]: Please update copyright.

**Pull request merged**